big-practice/src/javascript/models/book.model.js

```diff
@@ -0,0 +1,21 @@
+import { BookHelper } from '../helpers/book.helper';
+export class Model {
+  constructor() {
+    this.bookList = [];
+    this.bookHelper = new BookHelper();
+  }
+
+  async getBookList() {
+    this.bookList = await this.bookHelper.getRequest(`/books`) || [];
```

Comment:
Should use try catch here. What will happen if this function can not respond or return an error?